README.md

```diff
@@ -0,0 +1,9 @@
+# Wordpress Bedrock Integrations
+
+[![Unsupported](https://img.shields.io/badge/Pantheon-Unsupported-yellow?logo=pantheon&color=FFDC28)](https://pantheon.io/docs/oss-support-levels#unsupported)
+
+## About
+This repo contains settings, configuration, and functionality to enable a WordPress site to work on Pantheon. 
```

Comment:
We should clarify the upstream this is intended to be used with (`wordpress-bedrock-recommended`), and that it is [included in the upstream](https://github.com/pantheon-systems/wordpress-bedrock-recommended/blob/master/composer.json#L25)  by default.

And maybe we call out how this repo is being used within `wordpress-bedrock-recommended` (e.g. This repo is pulled into Pantheon's WordPress Bedrock Recommended upstream, and its contents are placed into the necessary places via the [cleanup-composer script](https://github.com/pantheon-systems/wordpress-bedrock-recommended/blob/master/scripts/composer/cleanup-composer#L13)). We should also document the cleanup-composer script in the upstream, but adding a note about it here should give folks some context as to what purpose this repo serves.